target/linux/ramips/image/mt7621.mk

```diff
@@ -317,9 +328,20 @@ define Device/dlink_dir-882-a1
   DEVICE_MODEL := DIR-882
   DEVICE_VARIANT := A1
   DEVICE_PACKAGES += kmod-usb3 kmod-usb-ledtrig-usbport
+  IMAGE/factory.bin := append-kernel | append-rootfs | check-size | \
+	  sign-dlink-ru 57c5375741c30ca9ebcb36713db4ba51 \
+	  ab0dff19af8842cdb70a86b4b68d23f7
```

Comment:
That's added to the wrong device?

Anyway, I'd make the factory image stuff a separate commit ...

Reply:
Whoops! Moved to the right section.

I moved the factory image stuff to a separate commit now too.